migrations/init_database.sql

```diff
@@ -0,0 +1,58 @@
+create table profile
```

Comment:
Somewhere you use capslock for CREATE, somewhere you don't. Please use the same typographical convention for all special words like CREATE, OR, REPLACE, UPDATE, IF, NOT, EXIST etc.